api/health/index.ts

```diff
@@ -37,39 +34,28 @@ export const healthChecks = {
   },
 }
 
-/*if (showFeature(FEATURE_TERMS_AND_CONDITIONS_ENABLED)) {
-  healthChecks.checks = {...healthChecks.checks, ...{
-  termsAndConditions: checkServiceHealth(getConfigValue(SERVICES_TERMS_AND_CONDITIONS_URL)),
-    },
-  }
-}*/
-
-/**
- * Add Reform standard Health Checks for /health and /health/liveness
- * to the application.
- *
- * Note that these checks are hit in the Build pipelines.
- *
- * @param app
- */
 export const addReformHealthCheck = app => {
-
+  if (showFeature(FEATURE_TERMS_AND_CONDITIONS_ENABLED)) {
+    config.checks = {
+      ...config.checks, ...{
+        termsAndConditions: checkServiceHealth(getConfigValue(SERVICES_TERMS_AND_CONDITIONS_URL)),
+      },
+    }
+  }
   if (showFeature(FEATURE_REDIS_ENABLED)) {
     xuiNode.on(SESSION.EVENT.REDIS_CLIENT_READY, (redisClient: any) => {
-      app.locals.redisClient = redisClient
-      healthChecks.checks = {...healthChecks.checks, ...{
-          redis: healthcheck.raw(async () => {
-            const status = await redisHealth()
-            return status ? healthcheck.up() : healthcheck.down()
+      console.log('REDIS EVENT FIRED!!')
+      config.checks = {...config.checks, ...{
+          redis: HealthCheck.raw(() => {
+            return redisClient.connected ? HealthCheck.up() : HealthCheck.down()
           }),
-        }}
-
-      healthcheck.addTo(app, healthChecks)
+        },
+      }
     })
     xuiNode.on(SESSION.EVENT.REDIS_CLIENT_ERROR, (error: any) => {
       logger.error('redis Client error is', error)
     })
-  } else {
-    healthcheck.addTo(app, healthChecks)
   }
+  console.log('config', config)
```

Comment:
use the logger instead of console.log is needed

Reply:
okay